src/util.ts

```diff
@@ -1,3 +1,4 @@
+import { ProfileId } from '@superfaceai/cli/dist/common/profile';
```

Comment:
ProfileId (and MapId) helper will be moved to AST to enable using them in more of our projects. I will try to do release today :)

Reply:
@Jakub-Vacek may I merge this now and import ProfileId helper from AST later?

Reply:
@janhalama definitely :)